foundation/settings.py

```diff
@@ -224,11 +240,27 @@ def _parse_email_list(varname):
     }
 
 # Database configuration
+# (optional) Create your local DB
+"""
+CREATE USER okfn WITH PASSWORD 'okfn';
+ALTER ROLE okfn SUPERUSER;
+CREATE DATABASE okfn OWNER okfn;
+"""
```

Comment:
Is this meant to be run always/everywhere? or just locally

Reply:
Removed, this is not necessary.